.gitignore

```diff
@@ -56,5 +56,4 @@ build-iPhoneSimulator/
 # .rubocop-https?--*
 
 # data files
-books.json
-labels.json
\ No newline at end of file
+*.json
```

Comment:
As per this project we don't, since the `Storage` class you created by default saves `.json` in `./` and we cannot know the number of `.json` to be saved. Hence need to ignore all, are you can refactor the `Storage` to save the files in a specific folder.